source/sav/Sav.cpp

```diff
@@ -333,33 +334,55 @@ namespace pksm
         {
             return BadTransferReason::MOVE;
         }
-        else if (availableSpecies().count(pk.species()) == 0)
+        if (availableSpecies().count(pk.species()) == 0)
```

Comment:
Why did you remove these `else`s?

Reply:
They didn't have to be removed per se, but removing them means that if the function has to be maintained more, then potential false negatives from skipped evaluations can be avoided. Consider the abilities check, for example.

Reply:
Ah, fair enough, makes sense